modules/webgl2-polyfill/src/polyfill-context.js

```diff
@@ -185,17 +39,38 @@ function initializeExtensions(gl) {
   }
 }
 
+// Install simple overrides (mostly get* functions)
+function installOverrides(gl, {target, target2}) {
```

Comment:
Just moving functions around to keep exported function at the top, simplify reading the code.